### PR TITLE
Issue 259 introduce release

### PR DESCRIPTION
**Pull request by MALPI**
- Introduce a manual step for the iteration of versions
- Set environment variables for gpg key and passphrase

### CODE REVIEW

**Comment by MALPI** [comment]
:+1:

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: release-to-maven-central
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release version'
+        required: true
+        default: '2.4.0'
```
Comment (otrosien) [inline .github/workflows/pre-release.yml:8]: what kind of defaults are these?

File: .github/workflows/pre-release.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      next_version:
+        description: 'Next version'
+        required: true
+        default: '2.4.0'
```
Comment (otrosien) [inline .github/workflows/pre-release.yml:12]: ?

**Comment by otrosien** [comment]
👍

**Comment by MALPI** [comment]
:+1:

**Pull request merged**